tests/test_package_generation.py

```diff
@@ -0,0 +1,61 @@
+"""Checks that the cookiecutter works."""
+
+import pathlib
+import subprocess
+from typing import Callable
```

Comment:
```suggestion
import typing
```
then use `typing.Callable`. @p-j-smith persuaded me on this school of thought, so it's easy to see where something came from (i.e. `pathlib.Path`)

Reply:
For you, I will do this.

Reply:
It's for @p-j-smith more than me! But thank you